### ISSUE TITLE
CN Security - Supply Chain through to Runtime

### ISSUE THREAD

**Issue by sonnambulo** [cloudnativemcr/speakers#6]
* **Title**:  Mr
* **Speaker**: Jonathan Campbell
* **Type**: (Presentation 30-45 mins | Lightning Talk 5-10 mins) Presentation / Discussion
* **Level**: (basic | standard | advanced) TBC
* **Tags**: (e.g. kubernetes, devops, live-coding, workshop) Open Source, Devops, Security, Kubernetes.

**Description**
Hi, the content for this will be based on Open Source security tools eg Trivy for Scanning - from Supply Chain (Code) to Runtime. This content has not been written or finaliased at this time - wanted to know if there would be an appetite for this type of discussion.

**Speaker Bio** (optional)

My name is Jonathan Campbell. I am an EMEA Solution Architect at Aqua Security, based in the north of England. I help enterprises gain visibility and secure their cloud native applications and infrastructure from Code to Runtime.

I have almost 20 years presales experience witha focus on Security and have previously worked at security vendors covering endpoint, traditional network, public and cloud native security.

**Comment by dabukalam** [comment]
@sonnambulo we're planning a meetup on the 21st March - does that evening work for you?

**Comment by sonnambulo** [comment]
Hi Danny,

Many thanks for getting back to me. Unfortunately this date doesn’t work for me – do you have a date in April or May planned yet?

Many thanks,

Jonathan

From: Danny Abukalam ***@***.***>
Date: Thursday, 16 February 2023 at 17:41
To: cloudnativemcr/speakers ***@***.***>
Cc: Jonathan Campbell ***@***.***>, Mention ***@***.***>
Subject: Re: [cloudnativemcr/speakers] CN Security - Supply Chain through to Runtime (Issue #6)

@sonnambulo<https://github.com/sonnambulo> we're planning a meetup on the 21st March - does that evening work for you?

**Comment by dabukalam** [comment]
Yes, planning another meetup in late May, so will let you know closer to the time. Thanks!

**Comment by dabukalam** [comment]
Hey Jonathan - I have a tentative date for the 30th May - would you be able to support that date?

**Comment by sonnambulo** [comment]
Hey Danny,

Many apologies for the delay in getting back to you. Totally missed this amongst the noise in my personal email inbox. I know it’s too late for this month and I am also unavailable.

Please can you let me know when the next slot might be?

Many thanks,

Jonathan

From: Danny Abukalam ***@***.***>
Date: Monday, 15 May 2023 at 10:49
To: cloudnativemcr/speakers ***@***.***>
Cc: Jonathan Campbell ***@***.***>, Mention ***@***.***>
Subject: Re: [cloudnativemcr/speakers] CN Security - Supply Chain through to Runtime (Issue #6)

Hey Jonathan - I have a tentative date for the 30th May - would you be able to support that date?

**Comment by yankcrime** [comment]
@sonnambulo It's been a while! We're trying to schedule the next one for late June or July, would you still be interested in coming along to give a talk?

**Comment by thomasriley** [comment]
Hi @sonnambulo. The Manchester community is active again, as you have likely seen. If you wish to talk at any point please do reach out. In the mean time, I will close this issue down as it is from some time again, please feel free to open back up if you're interested about speaking at a future event.
